### PR TITLE
Fix Revision Links settings page DPI

### PR DESCRIPTION
**Pull request by mdonatas**
## Proposed changes

- Fix DPI
- Align `Links` label to the `left`

This "bug" has made me both sad and angry.. look at the _200% before_ screenshot.. "drowners-donut" icon is not clickable. In order to get the link, I had to make a screenshot and use OneNote's OCR to copy the text! I honestly thought this was an incarnation of the most evil UX I've ever encountered....... turns out it's a DPI issue 😡

### Before

100%
revisionlinks_old_100

200%
revisionlinks_old_200

### After

100%
revisionlinks_new_100

200%
revisionlinks_new_200

## Test methodology 

- Manually with 100% and 200% scaling

## Test environment(s) 

- Windows 11 4K 200% scaling

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Review by mstv — APPROVED** [review]
LGTM, have not run

File: src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs [hunk 1]
```diff
@@ -228,21 +228,21 @@ private void InitializeComponent()
...
+            panel1.Size = new Size(1324, 18);
             panel1.TabIndex = 7;
             // 
             // label6
             // 
-            label6.Anchor = AnchorStyles.Left;
             label6.AutoSize = true;
             label6.ImeMode = ImeMode.NoControl;
-            label6.Location = new Point(206, -1);
+            label6.Location = new Point(0, 0);
```
Comment (gerhardol) [inline src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs:242]: Links label was probably intended to be above URI column (before there was a header?)
It does not add anything, remove?

**Review by gerhardol — COMMENTED** [review]
+1 have not run

File: src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs (hunk 1 above)
Comment (mdonatas) [reply src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs:242]: It does add some clarification esp. when starting fresh
image

**Comment by gerhardol** [comment]
> This "bug" has made me both sad and angry.. look at the 200% before screenshot.. "drowners-donut" icon is not clickable. In order to get the link, I had to make a screenshot and use OneNote's OCR to copy the text! I honestly thought this was an incarnation of the most evil UX I've ever encountered....... turns out it's a DPI issue 😡

The "life bouy" icon should be changed to the clickable icon used in other forms.

**Review by RussKie — APPROVED** [review]

**Review by RussKie — APPROVED** [review]

File: src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs (hunk 1 above)
Comment (gerhardol) [reply src/app/GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.Designer.cs:242]: I do not consider the label to add anything but am OK with keeping it

**Comment by mdonatas** [comment]
> > This "bug" has made me both sad and angry.. look at the 200% before screenshot.. "drowners-donut" icon is not clickable. In order to get the link, I had to make a screenshot and use OneNote's OCR to copy the text! I honestly thought this was an incarnation of the most evil UX I've ever encountered....... turns out it's a DPI issue 😡
> 
> The "life bouy" icon should be changed to the clickable icon used in other forms.

I'll do that in a separate PR if that's OK
image

**Pull request merged**